exonum_launcher/supervisor.py

```diff
@@ -71,16 +72,19 @@ def deinitialize(self) -> None:
         """Deinitializes the Supervisor by deinitializing the Protobuf Loader."""
         self._loader.deinitialize()
 
-    def _post_to_supervisor(self, endpoint: str, message: bytes, private: bool = True) -> List[str]:
-        message_data = message.hex()
-        data = json.dumps(message_data)
+    def _post_to_supervisor(self, endpoint: str, message: Optional[Message], private: bool = True) -> List[str]:
```

Comment:
Why private method should allow optional `message`? Wouldn't it be right to just not call it on the caller side if there is no message?